_client/src/Ui/Dialogs/DialogManager.cpp

```diff
@@ -14,6 +14,7 @@ DialogManager::~DialogManager()
     delete m_loginDialog;
     delete m_registrationDialog;
     delete m_messageDialog;
+    m_messageDialog->addButton(QMessageBox::Ok);
```

Comment:
WTF? Why in destructor? Did you test this?

Reply:
didn't see it. It had to be in initDialogs